platform/mds/mds/src/main/resources/db/migration/mysql/V1__Base_version.sql

```diff
@@ -457,7 +457,7 @@ INSERT INTO "TypeSettingOption" VALUES (1, 'REQUIRE'),
 INSERT INTO "TypeValidation" VALUES (1, 'mds.field.validation.minValue', 1),
                                     (2, 'mds.field.validation.maxValue', 1),
                                     (3, 'mds.field.validation.mustBeInSet', 2),
-                                    (4, 'mds.field.validation.cannotBeInSet', 1),
+                                    (4, 'mds.field.validation.cannotBeInSet', 2),
```

Comment:
please write new migration, do not change existing one. Moreover remember about postgres